core/src/main/java/org/arquillian/smart/testing/Configuration.java

```diff
@@ -2,27 +2,34 @@
 
 public class Configuration {
 
+    public static final String DEFAULT_MODE = "ordering";
+    public static final String DEFAULT_STRATEGIES = "";
+
     public static final String SMART_TESTING = "smart.testing";
     public static final String SMART_TESTING_MODE = "smart.testing.mode";
     public static final String SMART_TESTING_PLUGIN = "smart.testing.plugin";
```

Comment:
If you want to use `failsafe` or `surefire` for smart testing. By default both are modified.